aptos-move/framework/supra-framework/sources/stake.spec.move

```diff
@@ -437,7 +437,8 @@ spec supra_framework::stake {
 
     spec on_new_epoch {
         pragma verify = false; // TODO: set because of timeout (property proved).
-        pragma disable_invariants_in_body;
+        // TODO: Why we need to disable invariants in body? (SUPRA)
```

Comment:
I think Move prover will fail compilation if we do not have this pragma.

Reply:
This is not part of my change-set. it comes from this change-set f9652d1f0472fc60f887605ff13f779801b039e4